modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/PartitionReplicaLifecycleManager.java

```diff
@@ -1591,6 +1643,15 @@ private CompletableFuture<Void> stopAndDestroyPartition(ZonePartitionId zonePart
         ).thenApply(replicaWasStopped -> null);
     }
 
+    private CompletableFuture<Void> stopPartition(ZonePartitionId zonePartitionId, long revision) {
```

Comment:
The method stops the partition in a specific way ('for restart'), but its signature doesn't give a hint about it. It would be great if the name gave this hint, for example.

Reply:
done